src/app/core/services/media-parser.service.ts

```diff
@@ -17,7 +22,7 @@ export class MediaParserService {
     return tracks;
   }
 
-  extractTime(track: string) {
+  extractTime(track: string): any       {
```

Comment:
Please avoid using `any` as a return type. It would be better to use a more specific type that describes the structure of the expected return value. If the return type is not known or could be multiple types, consider using a TypeScript Union Type.